### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by pcunninghamatwombatsecurity**

### CODE REVIEW

File: _posts/2018-07-13-logging-maven-to-splunk.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+[Splunk](https://www.splunk.com/)
+
+Here are a few things that we considered doing with Splunk:
+- Highlight BUILD SUCCESS and BUILD FAILURE outcomes.
+- Watch builds in realtime using complex event filters.
+- Benchmark parts of the build process.
+- Create Splunk Alerts that respond to specific error messages.
+- Search specifically for warnings that are often ignored.
+
+Note: You don't specifically need to JSON to do these things with Splunk, but I've found that it makes constructing queries in Splunk more rewarding.
```
Comment (laurawhalin) [inline _posts/2018-07-13-logging-maven-to-splunk.md]: remove the word "to" before JSON

File: _posts/2018-07-13-logging-maven-to-splunk.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+Use Splunk's **New Search** to run the following query to see the results from your previous maven build:
+
+{% highlight plaintext %}
+index="maven"
+{% endhighlight %}
+
+### Time Traveling With Splunk
+
+Using a time-range of last 15 minutes is probably sufficient to view your most recent results, but now that it's in Splunk you can always go back to this time frame and view these results.
```
Comment (laurawhalin) [inline _posts/2018-07-13-logging-maven-to-splunk.md]: add "the" to "of last 15 minutes"

File: _posts/2018-07-13-logging-maven-to-splunk.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+### Building on Success
+
+You can easily search in Splunk to see if it was successful:
+
+{% highlight plaintext %}
+index="maven" msg="BUILD SUCCESS"
+{% endhighlight %}
+
+If you change your time range to use *Real-Time* (5 minute window works well), your **BUILD SUCCESS* event will appear as soon as the build is finished. No need to watch everything going by.
```
Comment (laurawhalin) [inline _posts/2018-07-13-logging-maven-to-splunk.md]: You're missing a second * after BUILD SUCCESS

File: _posts/2018-07-13-logging-maven-to-splunk.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+
+{% highlight plaintext %}
+index="maven" method="logStats" msg="Total time*"
+{% endhighlight %}
+
+From the events output by this search, you can find out the number of seconds each build took.
+
+### Looking for Errors and Warnings
+
+You might be watching a particularly long build for errors, or may need to find errors in the build you can a week ago:
```
Comment (laurawhalin) [inline _posts/2018-07-13-logging-maven-to-splunk.md]: The end of this needs slight adjustment. Maybe: "in the build you rand a week ago"?

**Comment by pcunninghamatwombatsecurity** [comment]
OK. Fixed all items tagged by Laura, cleaned up a few small things, and resubmitted.

**Pull request merged**